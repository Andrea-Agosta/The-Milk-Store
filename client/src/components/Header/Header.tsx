const Header = () => {
  return (
    <header className='text-[#ce89a1] font-title font-bold text-7xl text-center p-5'>THE MILK STORE</header>
  )
}

export default Header