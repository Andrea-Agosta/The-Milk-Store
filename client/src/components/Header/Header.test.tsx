import { render, screen } from '@testing-library/react';
import Header from './Header';


describe('Header component', () => {
  it('should render correctly', () => {
    render(<Header />);
    const container = screen.getByText(/the milk store/i)
    expect(container).toMatchSnapshot();
  });
});