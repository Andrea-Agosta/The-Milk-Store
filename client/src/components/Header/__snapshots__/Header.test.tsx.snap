// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<header
  class="text-[#ce89a1] font-title font-bold text-7xl text-center p-5"
>
  THE MILK STORE
</header>
`;
